cli/docker/docker.go

```diff
@@ -102,6 +102,19 @@ func ExecCommandIntoContainer(ctx context.Context, containerName string, user st
 
 	output = strings.ReplaceAll(output, "\n", "")
 
+	patterns := []string{
```

Comment:
```suggestion
        // Docker command garbage cleaner :D 
	patterns := []string{
```

Reply:
There is a trace log https://github.com/elastic/e2e-testing/pull/310/files#diff-0d15ecc74d96858feb62aab759545abeR114

Should we add the comment too?